### PR TITLE
Add wizard files to proposed structure

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,9 +91,13 @@ addons/<my_module_name>/
...
-`-- templates/
-    |-- <main_model>.xml
-    `-- <inherited_main_model>.xml
+|-- templates/
+|   |-- <main_model>.xml
+|   `-- <inherited_main_model>.xml
+`-- wizards/
+    |-- __init__.py
+    |-- <wizard model>.py
+    `-- <wizard model>.xml
```
Comment (moylop260) [inline CONTRIBUTING.md]: Can you change to snake_case from `wizard model` to `wizard_model`, please?

**Comment by moylop260** [comment]
I add this comment:
https://github.com/OCA/maintainer-tools/pull/73/files#r34901996

After this: :+1:

**Comment by bwrsandman** [comment]
Done. Please delete the branch after merging.

**Pull request merged**